packages/client/src/index.ts

```diff
@@ -1 +1,11 @@
-export default {};
+import {Property, Signature} from "../../message/src/index";
```

Comment:
Since this project has not been registered in npm yet, it cannot import types and functions through dependency.